server/alembic.ini

```diff
@@ -3,6 +3,7 @@ script_location = szurubooru/migrations
 
 # overriden by szurubooru's config
 sqlalchemy.url =
+revision_environment = true
```

Comment:
Not sure why this is needed?

Reply:
Initially I was getting errors related to modules not being found in my environment before I added this.